scripts/CMakeLists.txt

```diff
@@ -101,6 +101,47 @@ if (UPDATE_DEPS)
         include("${UPDATE_DEPS_DIR}/helper.cmake")
     endif()
 endif()
+
+# Configure compiler build options common to all targets
+function(lunarg_target_compiler_configurations TARGET_NAME WERROR_OPTION)
```

Comment:
What would be ideal is if we could actually share this code elegantly with the other repos that consume the utility libraries.

I think this can be done. I'll wait until this gets merged though.

Reply:
Yes that's what I was thinking!